tests/test_utils/mod.rs

```diff
@@ -1,6 +1,11 @@
 mod mock_database;
 mod types;
+
+// FIXME: workaround for false positive detection of unused_imports, which might be related to:
+// https://github.com/rust-lang/rust/issues/121708
+#[allow(unused_imports)]
```

Comment:
Ended up add this tentative workaround as the warning keeps coming up even after I specified all the import targets. (it disappears only if I remove `struct_loader.ts` file completely)